Gopkg.toml

```diff
@@ -0,0 +1,4 @@
+[[constraint]]
```

Comment:
please add yaml, cobra, go-homedir and go-glob as constrains

Reply:
okay I will do that

Reply:
Has this been done now? @dungeonmaster18